File: agent/bindings/csharp/AutocompleteResult.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class AutocompleteResult
  {
    [JsonProperty(PropertyName = "items")]
    public AutocompleteItem[] Items { get; set; }
    [JsonProperty(PropertyName = "completionEvent")]
    public CompletionBookkeepingEvent CompletionEvent { get; set; }
  }
}

File: agent/bindings/kotlin/lib/src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/AutocompleteResult.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class AutocompleteResult(
  val items: List<AutocompleteItem>,
  val inlineCompletionItems: List<AutocompleteItem>,
  val decoratedEditItems: List<AutocompleteEditItem>,
  val completionEvent: CompletionBookkeepingEvent? = null,
)

